tests/search/reindexing/reindexing.rb

```diff
@@ -13,6 +13,51 @@ def setup
     set_owner('bjorncs')
   end
 
+  def test_add_const_field
+    set_description("test that adds a constant bool field to documents and verifies that the correct value is assigned during reindexing")
+    testdir = selfdir + "add_const_field/"
+
+    # First, we use the schema without the field const_bool
+    system("cp #{testdir}item.0.sd #{dirs.tmpdir}item.sd")
```

Comment:
This is one of the non-intuitive things about the system test framework. Running the test happens on the a `controller` node, while Vespa is running another node. Commands are run with RPC. Copying files needs to be done with something like:

```suggestion
    vespa.nodeproxies.first[1].copy_remote_file_to_local_file(selfdir + "/add_const_field/item.0.sd", dirs.tmpdir + "item.sd")
```

the same goes for the other line using `cp`

Reply:
I took this part from [schemachanges_struct_attributes.rb](https://github.com/vespa-engine/system-test/blob/master/tests/search/schemachanges/schemachanges_struct_attributes.rb) (and [schemachanges_base.rb](https://github.com/vespa-engine/system-test/blob/master/tests/search/schemachanges/schemachanges_base.rb)). `vespa.nodeproxies.first[1].copy_remote_file_to_local_file` isn't used there, is it? Why do these tests work?